WordCounter/WordCounter.cs

```diff
@@ -8,22 +8,63 @@ namespace WordCounter
 {
     class WordCounter
     {
-        public WordCounter(string inputString)
+        private Dictionary<string, int> dict;
+
+        public WordCounter()
         {
-            Dictionary<string, int> dict = new Dictionary<string, int>();
+            dict = new Dictionary<string, int>();
+        }
 
-            char[] delimiters = { ' ', ',', '.' };
+        public void Start(string inputString)
+        {
+            //Note: some ? marks are still escaping somehow
+            char[] delimiters = { ' ', ',', '.','/','?','!','\'','\"',';',':','\r','\n' };
             string[] words = inputString.Split(delimiters);
-            for(int i = 0; i < words.Length; i++)
+            for (int i = 0; i < words.Length; i++)
             {
-                if (dict.ContainsKey(words[i])){
-                    dict[words[i]] += 1;
+                if (dict.ContainsKey(words[i].ToLower()))
+                {
+                    dict[words[i].ToLower()] += 1;
                 }
                 else
                 {
-                    dict.Add(words[i], 1);
+                    dict.Add(words[i].ToLower(), 1);
                 }
             }
         }
+
+        public Dictionary<string, int> getDictionary()
+        {
+            return dict;
+        }
+
+        public void Histogram()
```

Comment:
Histogram and WordCount are two different things.  This Method shouldn't be in this class.